src/Satooshi/Component/System/Git/GitCommand.php

```diff
@@ -39,9 +39,11 @@ public function getBranches()
      */
     public function getHeadCommit()
     {
-        $command = $this->createCommand("log -1 --pretty=format:'%H\n%aN\n%ae\n%cN\n%ce\n%s'");
+        $command = $this->createCommand("log -1 --pretty=format:'%H:%aN:%ae:%cN:%ce:%s'");
```

Comment:
According to https://git-scm.com/docs/pretty-formats
I think you can use '%n' for newline (I also actually confirmed it's working on my windows machine).

A concern with using ':' is that other parts may contain ':' and it will break something.